pkg/deploy/generator/resources.go

```diff
@@ -556,6 +558,65 @@ func (g *generator) lb() *arm.Resource {
 	}
 }
 
+func (g *generator) lbAlerts() []*arm.Resource {
+	return []*arm.Resource{
+		{
+			Resource: mgmtmonitor.ActionGroupResource{
+				ActionGroup: &mgmtmonitor.ActionGroup{
+					Enabled:        to.BoolPtr(true),
+					GroupShortName: to.StringPtr("rphealth"),
+				},
+				Name:     to.StringPtr("rphealth-ag"),
+				Type:     to.StringPtr("Microsoft.Insights/ActionGroups"),
+				Location: to.StringPtr("Global"),
+			},
+			APIVersion: apiVersions["monitor"],
+		},
+		{
+			Resource: mgmtmonitor.MetricAlertResource{
+				MetricAlertProperties: &mgmtmonitor.MetricAlertProperties{
+					Actions: &[]mgmtmonitor.MetricAlertAction{
+						{
+							ActionGroupID: to.StringPtr("[resourceId('Microsoft.Insights/ActionGroups', 'rp-lb-health-ag')]"),
+						},
+					},
+					Enabled:             to.BoolPtr(true),
+					EvaluationFrequency: to.StringPtr("PT5M"), //every 5min
+					Severity:            to.Int32Ptr(4),
+					Scopes: &[]string{
+						"[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]",
+					},
+					WindowSize:         to.StringPtr("PT15M"), //15min
+					TargetResourceType: to.StringPtr("Microsoft.Network/loadBalancers"),
+					AutoMitigate:       to.BoolPtr(true),
+					Criteria: mgmtmonitor.MetricAlertSingleResourceMultipleMetricCriteria{
+						AllOf: &[]mgmtmonitor.MetricCriteria{
+							{
+								CriterionType:   mgmtmonitor.CriterionTypeStaticThresholdCriterion,
+								MetricName:      to.StringPtr("DipAvailability"),
+								MetricNamespace: to.StringPtr("microsoft.network/loadbalancers"),
+								Name:            to.StringPtr("HealthProbeBelow99"),
+								Operator:        mgmtmonitor.LessThan,
+								Threshold:       to.Float64Ptr(99),
```

Comment:
We should be more aggressive from the gate.   99.95?

Reply:
Set to 99.95 over a 1h window.